assets/js/edd-checkout-global.js

```diff
@@ -184,9 +184,11 @@ window.EDD_Checkout = (function($) {
 
 						} else {
 
-							if (!inputs.is('.card-address-2')) {
-								inputs.attr('required','required');
-							}
+							inputs.each( function ( input ) {
+								if ( !$( input ).hasClass( '.card-address-2' ) ) {
```

Comment:
```suggestion
								if ( ! $( input ).hasClass( '.card-address-2' ) ) {
```

["When in doubt, space it out."](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#spacing)